tests/distributions/test_torch_patch.py

```diff
@@ -0,0 +1,16 @@
+import torch
+
+import pyro.distributions as dist
+from tests.common import requires_cuda
+
+
+@requires_cuda
+def test_dirichlet_grad_cuda():
+    concentration = torch.ones(3, requires_grad=True)
+    dist.Dirichlet(concentration).rsample().sum().backward()
+
+
+@requires_cuda
+def test_linspace():
+    x = torch.linspace(-1., 1., 100, device="cuda")
+    assert x.device == "cuda"
```

Comment:
This works correctly on the GPU, but we will need to test for `x.device.type == "cuda"` since device is a python object now.

Reply:
Done, let me know if there is anything else, or feel free to push to this branch.